wpilibj/src/test/java/edu/wpi/first/wpilibj/hal/MatchInfoDataTest.java

```diff
@@ -9,13 +9,33 @@
 
 import org.junit.jupiter.api.Test;
 
+import edu.wpi.first.hal.sim.mockdata.DriverStationDataJNI;
+import edu.wpi.first.wpilibj.DriverStation.MatchType;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
 class MatchInfoDataTest {
   @Test
-  @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
   void matchInfoDataDoesNotThrow() {
```

Comment:
You probably want to rename this test, now that it does more than simply check it doesn't throw.